addon/components/frost-radio-group.js

```diff
@@ -7,10 +7,15 @@ export default Ember.Component.extend(PropTypeMixin, {
   layout,
 
   propTypes: {
-    hook: PropTypes.string
+    hook: PropTypes.string,
+    id: PropTypes.string,
```

Comment:
Right now, `id` is tied into the updating of which button is in the checked state. This is driven from inside the controller.